dbt/parsely_dpl/README.md

```diff
@@ -0,0 +1,36 @@
+### Redshift dbt models for Parse.ly DPL
+
+---
+## Parse.ly DPL DBT
+A "data-build tool" for your Parse.ly data that automates SQL table creation for your data pipeline. It creates queryable tables for things like pageviews, x, and y, and handles the incremental loading of new data from S3 to your SQL tables. By handling this setup work for you, the data-build tool reduces configuration time and lets you get started writing your own custom queries more quickly.
+
+## Schemas/models
+pageviews
+videoviews
+campaigns
+content
+custom events
+sessions
+users
+
+## How to get started
+- Install DBT
+```pip install -r requirements.txt```
+- Edit the following files:
+-- ~/.dbt/profiles.yml (Input Redshift cluster and database information)
+-- parsely_raw_data/dbt/parsely_dpl/dbt_project.yml (Update all variables marked as configurable; details in document)
```

Comment:
Is it worth thinking about a very simple interactive script to set these variables? Doing that would let you explain each one as you go, instead of relying on the reader to find each thing that needs changed.